src/pages/report/assessment-report/StorageOverview.tsx

```
@@ -1,61 +1,176 @@
-      .filter((entry) => entry.count > 0)
-      .sort((a, b) => a.rangeStart - b.rangeStart)
-      .map(({ rangeStart, ...rest }) => rest);
-  }, [data, minValue, step]);
+      });
+  }, [DiskSizeTierSummary, viewMode]);
+  
+  const onDropdownToggle = () => {
+    setIsDropdownOpen(!isDropdownOpen);
+  };
+  
+  const onSelect = (event: React.MouseEvent<Element, MouseEvent> | undefined, value: string | number | undefined) => {
```

ESLint

[error] 'event' is defined but never used. Allowed unused args must match /^_/u.

(@typescript-eslint/no-unused-vars)